yugabyte/debezium-connector-yugabytedb
# Add test for tablet split after consistent stream creation

### Testing

Added new test `snapshotShouldBeCompletedOnParentIfSplitHappenedAfterStreamCreation` to test the following scenario between tablet split & consistent snapshot:

1. Insert 10 rows for table t1.
2. Consistent snapshot Stream created
3. Split tablet
4. Insert 5 more rows
5. Deploy connector (GetTabletListToPollForCDC should receive only parent tablet)
6. Assert 10 rows received in snapshot & 5 rows in streaming

Following things are verified:
1. Snapshot takes place on the parent tablet 
2. Any DMLs performed after the split are not part of snapshot records.
